jest/babel-preset-vite/babel-plugin-syntax-vite-meta-env.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+"use strict";
+
+const template = require("@babel/template").default;
+
+/**
+ * Add import.meta.env support
+ */
+module.exports = function viteMetaEnvBabelPlugin() {
+  const ast = template.ast(`
+  ({
+    env: {
+      ...Object.fromEntries(
+        Object.entries(process.env).filter(([k]) => /^VITE_/.test(k)),
+      ),
+      MODE: process.env.MODE || 'test',
+      NODE_ENV: process.env.NODE_ENV || 'test',
+      DEV: process.env.DEV ? process.env.DEV != false : true,
+      PROD: process.env.PROD ? process.env.PROD != false : false,
```

Comment:
just looking at this now, I don't think this is right.  `process.env` values will be `string | undefined` AFAIK.  My intention was that if the are set, they are considered true unless they were explicitly `false`.  I think they should be:

```suggestion
      DEV: process.env.DEV ? process.env.DEV !== 'false' : true,
      PROD: process.env.PROD ? process.env.PROD !== 'false' : false,
```

Reply:
Actually I'm half tempted to use `process.env.NODE_ENV` instead

```suggestion
      DEV: process.env. NODE_ENV ? process.env.NODE_ENV !== 'production' : true,
      PROD: process.env. NODE_ENV ? process.env.NODE_ENV === 'production' : false,
```

Reply:
I went even further and now derive all the standard env variables from `NODE_ENV` if it is provided.